spec/getVendorsByReferenceSpec.js

```diff
@@ -0,0 +1,99 @@
+"use strict";
+
+const getVendorsByReference = require('../lib/actions/getVendorsByReference.js');
+const orderWebhook = require('./fixtures/orderWebhook.json');
+const getVendorsByReferenceResponse = require('./fixtures/getVendorsResponse.json');
+const elasticio = require('elasticio-node');
+const filterLeadTimes  = require('../lib/getVendorsByReference/filterLeadTimes.js');
+const messages = elasticio.messages;
+// disable requests to the outside world
+const nock = require("nock");
+// nock.recorder.rec();
+nock.disableNetConnect();
+// silence console logs
+console.log = () => {};
+
+describe("Retrieve vendors lead times", () => {
+
+  beforeEach(() => {
+    this.message = {
+      body: orderWebhook
+    }
+
+    this.config = {
+      host:   "https://vendors-staging.herokuapp.com",
+      apiKey: "c91966a95dfdf3f6a446dedc189f49c75827e20f84b30c24d22749810695d86c"
```

Comment:
We might want to replace these credentials with some dummy data given that we are mocking everything out with nock.